apps/pie-docs/src/__tests__/_11ty/shortcodes/card.test.js

```diff
@@ -0,0 +1,25 @@
+const card = require('../../../_11ty/shortcodes/card');
+
+const item = {
+    linkText: 'label',
+    href: 'link.com',
+};
```

Comment:
is it worth us adding a test for the target attribute? I know default for `shouldOpenInNewTab` is false but wonder if it's worth covering